### ISSUE TITLE
Consider adding stretching overscroll from Android 12

### ISSUE THREAD

**Issue by nt4f04uNd** [nt4f04uNd/sweyer#96]
Now that Flutter's stretching overscroll seems really on-par with the native implementation, we can consider enabling it from the relevant (or all) API levels

**Comment by Abestanis** [comment]
Yeah, sounds like a good idea to keep consistency with the rest of the system. Lets only enable it for API 31 and above.
